index.js

```diff
@@ -14,6 +14,9 @@ const events = [
     "pointermove",
     "pointerrawupdate",
     "pointerup",
+    "jstodefpress",
+    "jstodefmove",
+    "jstodefrelease",
```

Comment:
Similar detached event pattern as `defold` prepended events to track and visualize them separately.